### DISCUSSION TITLE
Will the  delete and reboot service calls be integrated into the card?

### DISCUSSION THREAD

**Discussion by derandiunddasbo** [DurgNomis-drol/google_home_timers_card#16] [Ideas]
As you contributed the delete/reboot service calls for ha-google-home 1.3.0 I suspect, you are already working on integrating them into the google_home_timers_card as well, aren't you?

Just wanted to know if it's worth to mess around with button cards and stuff, if an integration into this card is already under development.

**Comment by DurgNomis-drol** [comment]
I have not started yet. BUT i need to rewrite the card to make it take advantage of the new features in `ha-google-home`. This will also fix #17. Maybe this weekend i get to do it. 😊 if you have any other suggestions, you are more then welcome to add them.

Contributions are also more then welcome 😁
